main.py

```diff
@@ -0,0 +1,113 @@
+import sqlite3
+import random
+import threading
+import os
+import dbActions
+import Settings
+
+
+def finish_game(name, level):
+    """
+    inserts player's name and score, and finishes the game.
+    :param name: player's name
+    :param level: player's score
+    """
+    con = raise_database()
+    dbActions.insert_new_score(name, level, con, Settings.db_table_name)
+    print "game over"
+    os._exit(1)
+
+
+def calculate_result(first_num, second_num, operation):
+    """
+    calculates equation result
+    :param first_num
+    :param second_num
+    :param operation
+    :return: math result
+    """
+    if operation == '/':
+        return first_num / second_num
+    elif operation == '*':
+        return first_num * second_num
+    elif operation == '+':
+        return first_num + second_num
+    elif operation == '-':
+        return first_num - second_num
+
+
+def check_no_zero(operation, second_num, level):
```

Comment:
are you talking about case where the first number is less than the second? because in this case I just except from the player to insert zero as an answer